src/cmd-buildpost-ec2

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3 -u
+# An operation that mutates a build by uploading to EC2,
+# extending the meta.json with AMI information.
+
+import os,sys,json,yaml,shutil,argparse,subprocess,re,collections
+import tempfile,hashlib,gzip
+
+def run_verbose(args, **kwargs):
+    print("+ {}".format(subprocess.list2cmdline(args)))
+    subprocess.check_call(args, **kwargs)
+
+# Parse args and dispatch
+parser = argparse.ArgumentParser()
+parser.add_argument("--build", help="Build ID",
+                    required=True)
+parser.add_argument("--region", help="EC2 region",
+                    required=True)
+parser.add_argument("--bucket", help="S3 Bucket",
+                    required=True)
+parser.add_argument("--tag", help="AMI tag in key=value format",
+                    default=[], action='append')
+args = parser.parse_args()
+
+with open('src/config/manifest.yaml') as f:
+    manifest = yaml.safe_load(f)
+
+base_name = manifest['rojig']['name']
+ami_name_version = f'{base_name}-{args.build}'
+
+builddir = f'builds/{args.build}'
+buildmeta_path = f'{builddir}/meta.json'
+with open(buildmeta_path) as f:
+    buildmeta = json.load(f)
+
+tmpdir='tmp/buildpost-ec2'
+if os.path.isdir(tmpdir):
+    shutil.rmtree(tmpdir)
+os.mkdir(tmpdir)
+
+# Copied from prune_builds
+def write_json(path, data):
```

Comment:
Should be able to factor this and `run_verbose` out to a `cmdlib.py` file and just `import cmdlib`.